src/components/Collections/Table/Table.vue

```diff
@@ -292,6 +293,20 @@ export default {
   },
   methods: {
     ...mapActions("sync", ["startLoading"]),
+    searchIfEnter(event) {
+      /**
+       * detect if enter has been pressed after entering
+       * a text to search
+       */
+      // check if the key pressed is the enter key
+      if (event.key === "Enter" || event.keyCode === 13) {
+        /**
+         * event.key is the modern way of detecting keys
+         * event.keyCode is deprecated (left here for for legacy browsers support)
+         */
+        if (this.searchString != "") this.search();
```

Comment:
I think we should also check for leading or trailing spaces here and if they're present, trim those before proceeding with this check

Reply:
makes sense